spec/support/shared_examples/simple_user_is_not_authorized_example.rb

```diff
@@ -0,0 +1,19 @@
+RSpec.shared_examples 'simple user is not authorized' do
```

Comment:
You can add one test for Admin::AdminController where you just make a request on one of the existing pages and do not run these tests for each admin page because it's 3 additional tests for each page which you will add in future